modules/images/webp-uploads/load.php

```diff
@@ -154,6 +180,24 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 				continue;
 			}
 
+			$original_thumbnail_image_filesize = isset( $properties['filesize'] ) ? (int) $properties['filesize'] : 0;
+			$webp_thumbnail_image_filesize     = isset( $source['filesize'] ) ? (int) $source['filesize'] : 0;
+			if ( $original_thumbnail_image_filesize > 0 && $webp_thumbnail_image_filesize > 0 ) {
+				/** This filter is documented in modules/images/webp-uploads/load.php */
+				if (
+					apply_filters( 'webp_uploads_discard_larger_generated_images', true )
+					&& $webp_thumbnail_image_filesize >= $original_thumbnail_image_filesize
+				) {
+					$destination = path_join( $original_directory, $source['file'] );
+					if ( ! file_exists( $destination ) ) {
+						continue;
+					}
```

Comment:
Same question as above.

Reply:
Thanks for pointing it out. Changes were made to the PR.